book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+    rental.book = self
```

Comment:
- [ ] Great job for adding the method but I think it would be great if you add the right association `qne achieving the 'has-many' association` here as well since this method it will be between the person and date in book file so you need to pass a person and date as a parameter  then you can create a new Rental with Rental.new and pass to this new `(person, book, self)` 🤩✨💯🔥 you can check this [example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more explaination